### DISCUSSION TITLE
Issue with New Related Content and Upcoming Content

### DISCUSSION THREAD

**Discussion by meitedaf** [Cyfrin/foundry-full-course-cu#2156] [Ideas]
Hi Cyfrin Team and our dear Patrick,

I’ve noticed that the videos listed under New Related Content in the [content-updates.md](https://github.com/Cyfrin/foundry-full-course-cu/blob/main/content-updates.md) are no longer available. 

I’ve tried searching for these videos on YouTube, but it seems they haven’t been released. there are several items in the Upcoming Content section that sound incredibly useful and intriguing, such as:

	•	Building a Charismatic Web3 Resume
	•	DIY Blockchain Time!
	•	The Fresher's Guide to a Web3 Internship/Job

These items were listed over 10 months ago. I am curious about what might have delayed their release. Is there anything that’s holding you back?

We appreciate all your hard work and look forward to seeing these videos soon.

Thank you, and I hope to hear from you soon.

**Comment by EngrPips** [comment]
Hello @meitedaf, Thanks for the comment, and I trust this will be attended to by @PatrickAlphaC  as soon as he is available.

**Reply by PatrickAlphaC** [reply]
Hello! Yes! This would make an awesome article, I don't think a video would really make sense here. Let me give you some tips now, and then we can see if @cromewar made this content. I think he was planning on making a blog on this!

# Building a Charismatic Web3 Resume

TL;DR you have 2 paths:

1. `Smart contract developer`: If you want to go this path, what matters is your GitHub. How many cool projects have you done? Are you contributing to projects? Trying to fix issues in projects that you admire? How many hackathons have you attended? If you get some hackathon results under your belt, that's your starting point.

For beginners, you can start with jobs like:
- Junior technical content
- Junior documentation
- Junior developer
- Teacher's assistant (like our very own @EngrPips!!)
etc. 

Many times docs writer end up being developers later! You can of course, always go straight for developer roles as well. 

2. `security researcher`: The path here is more straightforward but more competitive. You barely need a resume, all you need is results! Compete on platforms like [codehawks](https://codehawks.com/) or win bug bounties on [Immunefi](https://immunefi.com/) and you will get offers, trust me. Security is in crazy demand, but you have to be good! 

# DIY Blockchain Time!

This might be a course in 2025 or 2026... But, idk how important it is for web3. There is a LOT that is involved here. Stay tuned! 

# The Fresher's Guide to a Web3 Internship/Job

See above :)

**Reply by EngrPips** [reply]
I remain my humble self, sir, 🙇🏾‍♂️

**Reply by meitedaf** [reply]
Hi Patrick,

Thank you for your detailed response and for sharing those insightful tips! I really appreciate the guidance on building a charismatic Web3 resume and the paths for both smart contract developers and security researchers. 

A special thanks to @EngrPips, who has really helped me a lot in my learning journey.

I’m looking forward to any upcoming articles or blog posts you and your team might release. Thanks again for all the hard work and dedication you put into making such useful content. I’m excited to see what comes next!

**Reply by EngrPips** [reply]
You are welcome, Friend. We rise by lifting others.
